src/function/arithmetic/gcd.js

```diff
@@ -89,6 +93,28 @@ export const createGcd = /* #__PURE__ */ factory(name, dependencies, ({ typed, m
     }
   )
 
+  /**
+ * Calculate gcd for numbers
+ * @param {number} a
+ * @param {number} b
+ * @returns {number} Returns the greatest common denominator of a and b
+ * @private
+ */
+  function _gcdNumber (a, b) {
+    if (!isInteger(a) || !isInteger(b)) {
+      throw new Error('Parameters in function gcd must be integer numbers')
+    }
+
+    // https://en.wikipedia.org/wiki/Euclidean_algorithm
+    let r
+    while (b !== 0) {
+      r = _modNumber(a, b)
```

Comment:
Yeah, it would be

Reply:
Injecting `mod` there would cause the functionality to break because negative values can becomee the divisor when calculating `gcd`. But in the case of `mod`, negative divisors are not allowed.

Reply:
Would it be an idea to change `mod` to support negative divisors?